client/config.go

```diff
@@ -21,11 +21,11 @@ type Config struct {
 func (c Config) Example() string {
 	return ` configuration {
   // Optional. if you want to assume role to multiple account and fetch data from them
-  // accounts "<YOUR ACCOUNT ID>" {
+  // accounts "<UNIQUE ACCOUNT IDENTIFIER>" {
     // Optional. Role ARN we want to assume when accessing this account
     // role_arn = < YOUR_ROLE_ARN >
-    // Optional. Account ID we want to assume when accessing this account - override the block label
-    // account_id = < YOUR ACCOUNT ID >
```

Comment:
why? its still a valid attribute in the hcl, I am confused from the previous PR #396

Reply:
The mistake is that it is still a valid attribute in hcl. fixed now

Reply:
We had already made it an error: https://github.com/cloudquery/cq-provider-aws/blob/2fc0a4d434fb274352cb1c74562b22fc8adb8713/client/client.go#L361-L363